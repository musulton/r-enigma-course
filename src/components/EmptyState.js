import Styled from "styled-components";

const H3 = Styled.h3`
    margin: 20px 0;
`

const EmptyList = (props) => {
    return (
        <H3>{props.text}</H3>
    )
}

export default EmptyList;
