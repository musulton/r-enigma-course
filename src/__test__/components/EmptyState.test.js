import EmptyState from "../../components/EmptyState";
import {render} from "@testing-library/react";

describe("EmptyState component", () => {
  test("should render correctly", () => {
    const {asFragment} = render(<EmptyState text="Data is empty" />)

    expect(asFragment()).toMatchSnapshot()
  })
})