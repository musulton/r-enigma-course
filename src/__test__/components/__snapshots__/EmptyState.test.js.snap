// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyState component should render correctly 1`] = `
<DocumentFragment>
  <h3
    class="sc-bgqQcB bgrhhs"
  >
    Data is empty
  </h3>
</DocumentFragment>
`;
